src/draw/sw/lv_draw_sw_transform.c

```diff
@@ -278,7 +278,11 @@ void lv_draw_sw_transform(lv_draw_unit_t * draw_unit, const lv_area_t * dest_are
                 break;
 #endif
             default:
-                break;
+                LV_LOG_WARN("Color format 0x%02X is not enabled. "
```

Comment:
So what happens in runtime if this condition is reached? If something is not a supported / enabled feature, is an invalid condition, would not a LOG_ERR to be better?

Reply:
Usually we use LOG_ERROR on crashes when you can't use the system anymore. Like asserts, out of memory, etc. In this case the screen will be scrambled, which is not a critical error.